.vsts-ci.yml

```diff
@@ -8,59 +11,70 @@ resources:
+    PB_PublishBlobFeedUrl: https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
 
 phases:
 - template: /eng/build.yml
   parameters:
     agentOs: Windows_NT
     queue:
-      ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+      ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
         name: Hosted VS2017
-      ${{ if notIn(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:        
+      ${{ if ne(variables['Build.Reason'], 'PullRequest') }}:
         name: DotNetCore-Build
         demands:
         - agent.os -equals Windows_NT        
       parallel: 2
       matrix:
-        Build_Debug:
-          _BuildConfig: Debug
-          _PublishType: none
-          _SignType: test
+        ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+          Build_Debug:
+            _BuildConfig: Debug
+            _PublishType: none
+            _SignType: test
         Build_Release:
           _BuildConfig: Release
-          ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+          ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
             _PublishType: none
             _SignType: test
-          ${{ if notIn(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+          ${{ if ne(variables['Build.Reason'], 'PullRequest') }}:
             _PublishType: blob
             _SignType: real
 
-
-- template: /eng/build.yml
-  parameters:
-    agentOs: Linux
-    dockerImage: microsoft/dotnet-buildtools-prereqs:ubuntu-14.04-cross-0cd4667-20170319080304
-    queue: 
-      ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+- ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+  - template: /eng/build.yml
+    parameters:
+      agentOs: Linux
+      dockerImage: microsoft/dotnet-buildtools-prereqs:ubuntu-14.04-cross-0cd4667-20170319080304
+      queue: 
         name: DotNetCore-Linux
-      ${{ if notIn(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:        
-        name: DotNetCore-Test
-        demands:
-          - agent.os -equals Linux
-      parallel: 2
-      matrix:
-        Build_Debug:
-          _BuildConfig: Debug
-          _PublishType: none
-          _SignType: test
-        Build_Release:
-          _BuildConfig: Release
-          _PublishType: none
-          ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+        parallel: 2
+        matrix:
+          Build_Debug:
+            _BuildConfig: Debug
+            _PublishType: none
+            _SignType: test
+          Build_Release:
+            _BuildConfig: Release
+            _PublishType: none
             _SignType: test
-          ${{ if notIn(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
-            _SignType: real
 
+  - template: /eng/build.yml
+    parameters:
+      agentOs: MacOS
```

Comment:
Change this to Darwin.  VSTS names mac os 'Darwin'.  For consistency, we tend to follow suit.  If anyone tries to use the run-time variable 'Agent.OS', that's the value they'll get.

Reply:
Done.